phys2bids/interfaces/txt.py

```diff
@@ -9,34 +9,27 @@
 from phys2bids.physio_obj import BlueprintInput
 
 
-def populate_phys_input(filename, chtrig):
+def labchart_read(channel_list, chtrig, header=[]):
     """
-    Populate object phys_input
-    for now this works only with labchart files
-        Input (Properties)
-    ------------------
-    filename: str
-        path to the txt labchart file
+    Reading function for labchart files
+    Parameters
```

Comment:
Valid for all docstrings:
I'm doubting that an empty line is needed before the "title" (e.g. Parameters). At least, I see numpy functions having them:
https://github.com/numpy/numpy/blob/d9b1e32cb8ef90d6b4a47853241db2a28146a57d/numpy/lib/function_base.py#L1284
Did you try to import this function in Spyder and read its help? Or make an html page with sphinx?
Please check that it's ok to not have an empty line before all the titles.

Also, please add **Raises** sections (https://numpydoc.readthedocs.io/en/latest/format.html), please?